examples/two_dimensional_turbulence.jl

```diff
@@ -47,14 +49,37 @@ vᵢ .-= mean(vᵢ)
 
 set!(model, u=uᵢ, v=vᵢ)
 
-simulation = Simulation(model, Δt=0.2, stop_time=50)
+# ## Simulation
+#
+# Next we construct a simulation. To choose an appropriate timestep we check the
+# CFL number. In this case, the initial flow is highly variable and given
+# that there is no forcing in the model we expect flow to smoothen with time.
+# Thus, the CFL that corresponds to the initial time should be a close upper bound.
+#
+# For the `:RungeKutta3` timestepper used in this model, a timestep `Δt` that
+# gives CFL ⪅ 1 should be adequate; the `:QuasiAdamsBashforth2` requires CFL ⪅ 0.25.
+
+Δt = 0.03
+cfl = AdvectiveCFL(Δt)
+cfl(model)
```

Comment:
Maybe it's better to simply compute the time-step from the CFL.

Also, for the purpose of pedagogy, I think it is nice not to "tune" numbers too much. eg why not 0.01 or 0.05 rather than 0.03? When parameters appear tuned, they draw attention from the reader into the details and away from the bigger picture, which is more important. Perhaps choosing the time-step to be 0.5 or 0.2 of the initial CFL will help?

The instability we saw previously did not occur at the first time step though. So I think using the initial condition is rather conservative. Still, the physical explanation does make sense. We might want to mention that larger time-steps can be stable too.

Reply:
Ok. I resonate. 

Let’s just use the cfl-based criterion and adaptive timestep?

Reply:
I think we wanted to avoid the wizard to keep things simple. But I guess if we are computing CFL, maybe the point is moot. Maybe we want to encourage people to always use the wizard, too...

Reply:
In summary, using the wizard sounds good to me 🥲

Reply:
I also think we want to encourage using the wizard...